cypress/integration/quick-command-tests.js

```diff
@@ -20,7 +20,7 @@ describe('QuickCommand', () => {
 
     it('has a header with navlinks a logo', () => {
       cy.get('.logo').should('exist')
-        .get('a[data-cy=database-btn]').contains('DATABASE')
+        .get('a[data-cy=database-btn]').contains('PERSONNEL')
```

Comment:
This might get a little confusing since all these components start with 'database', but I like personnel better.